secboot/secboot_sb_test.go

```diff
@@ -1426,7 +1426,7 @@ func (s *secbootSuite) TestUnlockVolumeUsingSealedKeyIfEncryptedFdeRevealKeyErr(
 
 	opts := &secboot.UnlockVolumeUsingSealedKeyOptions{}
 	_, err := secboot.UnlockVolumeUsingSealedKeyIfEncrypted(mockDiskWithEncDev, defaultDevice, mockSealedKeyFile, opts)
-	c.Assert(err, ErrorMatches, `cannot unlock encrypted partition: cannot recover keys because of an unexpected error: cannot run fde-reveal-key "reveal": helper error`)
+	c.Assert(err, ErrorMatches, `cannot unlock encrypted partition: cannot recover keys because of an unexpected error: helper error`)
```

Comment:
(nitpick) I wonder if we should make the mocked error at the beginning of the function a bit more realistic now now that there is no error prefix anymore, so instead of `helper error` somehting like: `cannot run "fde-reveal-key": helper error`  (as system.Run() will generate an error like this) - mostly suggesting this so that it's clear from this test that in the failure case we generally can expect a useful error message.

Reply:
Done.